lib/blobs.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * @typedef {{
+ * blobId: string,
+ * blobIdBase64: string,
+ * objectId: string,
+ * status: string,
+ * startEpoch: number,
+ * endEpoch: number,
+ * size: number,
+ * timestamp: number
+ * }} Blob
+ */
+
+/**
+ * Returns a list of recent confirmed blobs.
+ *
+ * @param {typeof globalThis.fetch} fetch The fetch function to use.
+ * @returns {Promise<string[]>} A list of recent blob Base64 IDs.
+ */
+export const getRecentBlobs = async (fetch = globalThis.fetch) => {
+  const res = await fetch('https://walruscan.com/api/walscan-backend/testnet/api/blobs?page=0&sortBy=TIMESTAMP&orderBy=DESC&searchStr=&size=20')
+  const blobs = await res.json()
```

Comment:
Please check that `response.ok` is true. (Check out the "assert ok response" helper we use in other repositories.)

Reply:
Resolved in https://github.com/CheckerNetwork/walrus-checker/pull/12/commits/855db4cca55d56941dd55eea8f7f1740d9402d41